pkg/sqlite/migrations/38_scene_activity.up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE `scenes` ADD COLUMN `resume_time` float not null default 0;
+ALTER TABLE `scenes` ADD COLUMN `last_played_at` datetime not null;
```

Comment:
This causes an error during migration for existing systems as there is no default and is `not null`. It should accept and default to a null value.